hpat/hiframes/pd_series_ext.py

```diff
@@ -1270,23 +1270,18 @@ def typer(val1, val2):
 @overload(pd.Series)
 def pd_series_overload(data=None, index=None, dtype=None, name=None, copy=False, fastpath=False):
 
-    if index is not None:
-        def hpat_pandas_series_index_ctor_impl(
-                data=None,
-                index=None,
-                dtype=None,
-                name=None,
-                copy=False,
-                fastpath=False):
-            return hpat.hiframes.api.init_series(
-                hpat.hiframes.api.fix_df_array(data),
-                hpat.hiframes.api.fix_df_array(index),
-                name)
-
-        return hpat_pandas_series_index_ctor_impl
+    is_index_none = True if (isinstance(index, types.NoneType) or index is None) else False
```

Comment:
Will the following line work the same?
`is_index_none = isinstance(index, types.NoneType) or index is None`

Reply:
@densmirn Well, yes, I tend to avoid using python or this way though. But since we have Booleans exprs here anyway, I think I'll use shorter version. Thanks!